apps/webapp/src/components/swap/asset-out-selector.tsx

```diff
@@ -22,30 +22,25 @@ export const AssetOutSelector = ({ balances, setAssetOut, assetOut }: AssetOutSe
           </p>
         </div>
       </DialogTrigger>
-      <DialogContent className='max-w-[312px] bg-charcoal-secondary md:max-w-[400px]'>
-        <div className='relative z-10 flex flex-col gap-4 pb-5'>
```

Comment:
No need for the relative + absolute divs for the colorful background; that's now included by default